communication/slack-config/usergroups.yaml

```diff
@@ -8,16 +8,16 @@ usergroups:
       - release-management
       - sig-release
     members:
-      - aleksandra-malinowska
-      - bubblemelon
-      - calebamiles
-      - feiskyer
-      - hoegaarden
-      - idealhack
-      - justaugustus
-      - listx
-      - sumitranr
-      - tpepper
+      - aleksandra-malinowska # Patch Release Team
```

Comment:
Can we delegate this list to sig-release so it doesn't necessarily need slack admin approval?

(The technical answer is yes - tempelis does support this.)

Reply:
I had no clue; thanks for the tip. Fixed! :)